server/sonar-server/src/main/java/org/sonar/server/user/ws/SkipOnboardingTutorialAction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.user.ws;
+
+import org.sonar.api.server.ws.Request;
+import org.sonar.api.server.ws.Response;
+import org.sonar.api.server.ws.WebService;
+
+public class SkipOnboardingTutorialAction implements UsersWsAction {
+
+  @Override
+  public void define(WebService.NewController context) {
+    context.createAction("skipOnboardingTutorial")
+      .setPost(true)
+      .setInternal(true)
+      .setDescription("Stores that the user has skipped the onboarding tutorial and does not want to see it after future logins." +
+        " Calling this webservice several times will silently ignore subsequent requests.")
```

Comment:
Currently it works fine, if the user is not authenticated. I'd like to fix this in the ticket about the implementation of that ws. (Maybe this is even a good entry point for tdd ;) )